openstack/compute/v2/flavors/requests.go

```diff
@@ -21,22 +21,12 @@ type ListOptsBuilder interface {
 //
 // If no IsPublic argument is supplied in the request, OpenStack treats the FlavorType as Project.
 type FlavorType string
+
 const (
-        Project FlavorType =  "true"
-        Private FlavorType = "false"
-        All FlavorType = "None"
+	Project FlavorType = "true"
```

Comment:
I like `AccessType` and I the idea of appending "Access" to the names of all three types.  But `PublicAccess` can't be one of these types, because there is no API call that allows you to obtain all and only those flavors that are public.  

It really is the case that the Nova API gives you three options in your fetch: 

* all flavors, public or private, across all projects,
* private flavors across all projects, or 
* flavors available within a specific project (either because they are private to that project or because they are public).